scripts/sct_label_vertebrae.py

```diff
@@ -867,8 +867,14 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, verbose=1):
     :param verbose:
     :return:
     """
+
     # open segmentation
     seg = Image(fname_seg)
+
+    # change the orientation to RPI because the function assume dim[2] to be the Superior-to-Inferior axis
```

Comment:
shouldn't this comment now be changed for: "Change the orientation to RPI so that any orientation can be input"

Reply:
! Corrected! merci